browser/app/js/components/ObjectsList.js

```diff
@@ -19,78 +19,61 @@ import Moment from 'moment'
 import humanize from 'humanize'
 import connect from 'react-redux/lib/components/connect'
 import Dropdown from 'react-bootstrap/lib/Dropdown'
+import MaterialDesignIconicFonts from 'material-design-iconic-font/dist/css/material-design-iconic-font.min.css'
 
-let ObjectsList = ({objects, currentPath, selectPrefix, dataType, showDeleteConfirmation, shareObject, loadPath, checkObject, checkedObjectsArray}) => {
-  const list = objects.map((object, i) => {
-    let size = object.name.endsWith('/') ? '-' : humanize.filesize(object.size)
-    let lastModified = object.name.endsWith('/') ? '-' : Moment(object.lastModified).format('lll')
-    let loadingClass = loadPath === `${currentPath}${object.name}` ? 'fesl-loading' : ''
-    let actionButtons = ''
-    let deleteButton = ''
-    if (web.LoggedIn())
-      deleteButton = <a href="" className="fiad-action" onClick={ (e) => showDeleteConfirmation(e, `${currentPath}${object.name}`) }><i className="fa fa-trash"></i></a>
+let ObjectsList = ({objects, currentPath, selectPrefix, dataType, showDeleteConfirmation, shareObject, loadPath, checkObject, checkedObjectsArray, showObjectPreview, objectName}) => {
+    const list = objects.map ((object, i) => {
+        let size = object.name.endsWith ('/') ? '' : humanize.filesize (object.size)
+        let lastModified = object.name.endsWith ('/') ? '' : Moment (object.lastModified).format ('lll')
+        let loadingClass = loadPath === `${currentPath}${object.name}` ? 'fesl-loading' : ''
 
-    if (!checkedObjectsArray.length > 0) {
-      if (!object.name.endsWith('/')) {
-        actionButtons = <Dropdown id={ "fia-dropdown-" + object.name.replace('.', '-') }>
-                          <Dropdown.Toggle noCaret className="fia-toggle"></Dropdown.Toggle>
-                          <Dropdown.Menu>
-                            <a href="" className="fiad-action" onClick={ (e) => shareObject(e, `${currentPath}${object.name}`) }><i className="fa fa-copy"></i></a>
-                            { deleteButton }
-                          </Dropdown.Menu>
-                        </Dropdown>
-      }
-    }
+        let activeClass = ''
+        let isChecked = ''
+        let folderClass = dataType (object.name, object.contentType) == 'folder' ? 'objects__row--folder' : ''
 
-    let activeClass = ''
-    let isChecked = ''
 
-    if (checkedObjectsArray.indexOf(object.name) > -1) {
-      activeClass = ' fesl-row-selected'
-      isChecked = true
-    }
+        if (checkedObjectsArray.indexOf (object.name) > -1) {
+            activeClass = ' objects__row-selected'
+            isChecked = true
+        }
+
+
+        return (
+            <div key={ i } className={ "objects__row " + folderClass + loadingClass + activeClass } onClick={ (e) => showObjectPreview(e) }>
```

Comment:
@poornas This PR is to merge the ongoing UI changes to the newux branch of the upstream. There are more works and fixes still required to merge this in to the master.

Upload folder, Preview, Download and all the thumbnail related elements are only for UI purposes and there is no any functionality behind them